EstateReporting.Database/EstateReportingContext.cs

```diff
@@ -205,6 +213,8 @@
             {
                 await this.SetIgnoreDuplicates(cancellationToken);
             }
+
+            await this.CreateViews(cancellationToken);
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6325301)